gradle-language-server/src/main/java/com/microsoft/gradle/handlers/DependencyCompletionHandler.java

```diff
@@ -119,7 +119,8 @@
             completionItem.setKind(CompletionItemKind.Text);
             completionItem.setDetail("version");
           }
-          completionItem.setSortText(String.valueOf(i));
+          // Currently we have no more than 50 results, so the padding values should have at least 3 digits.
```

Comment:
> no more than 50 results

why 50?

Reply:
Ok I got it, it's from maven search API pagination.

Reply:
Yeah, I'll add some descriptions there to help to understand